scripts/postgresql-setup.sh

```diff
@@ -22,6 +22,14 @@ case "$(uname)" in
     force="-f"
   esac
 
+# This should work on all Linux variants as well as FreeBSD.
+numcores=$(getconf _NPROCESSORS_ONLN)
+if test "${numcores}" -le 2 ; then
+  numcores=1
+else
+  numcores=$(( numcores - 1 ))
+  fi
```

Comment:
```suggestion
fi
```
Alignment on the closing `fi`

Reply:
But the current indention matches the rest of the file.